starter-code/models/user.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+  facebookID: String
```

Comment:
The user model would probably need some more fields: username, password, role...